Sources/Player/Player+ControlCenter.swift

```diff
@@ -31,13 +31,22 @@ extension Player {
 
     func nowPlayingInfoMetadataPublisher() -> AnyPublisher<NowPlayingInfo, Never> {
         currentPublisher()
-            .map { current in
+            .map { [queuePlayer] current in
                 guard let current else {
                     return Just(NowPlayingInfo()).eraseToAnyPublisher()
                 }
-                return current.item.$asset
-                    .filter { !$0.resource.isLoading }
-                    .compactMap { $0.nowPlayingInfo() }
+                return Publishers.CombineLatest(current.item.$asset, queuePlayer.errorPublisher())
```

Comment:
Maybe we could use `itemUpdatePublisher` directly? We would be able to:

- Access the stored item list.
- Get the current `AVPlayerItem` and subscribe to its errors (without having to capture `queuePlayer`).

Reply:
Unfortunately, we can't use `itemUpdatePublisher` due to the transition from a loading asset to a loaded asset. By using `itemUpdatePublisher`, we are only listening to the transition between items and not between the different states of an item.